demos/horizontal_text_detection/python/horizontal_text_detection.py

```diff
@@ -175,18 +175,27 @@ def run(self):
 for thread in threads:
     thread.start()
 
+def finish():
+    global force_exit
```

Comment:
`force_exit` is defined outside of any function scope. 
Do we need `global` keyword? 

same applies to line **139**.